### PR TITLE
Login autofill optimizations

### PR DESCRIPTION
**Pull request by dev7ch**

### CODE REVIEW

File: modules/admin/src/views/layouts/nosession.php [hunk 1]
```diff
@@ -12,6 +12,8 @@
             <?php $this->beginBody() ?>
                 <?= $content; ?>
             <?php $this->endBody() ?>
+            <? $this->registerJS('$(window).load(function () {checkInputLabels(); });'); ?>
```
Comment (nadar) [inline modules/admin/src/views/layouts/nosession.php]: checkInputLabels() is already execute in the login.js script.

**Comment by nadar** [comment]
@dev7ch i just cleanup your code and the logic inside your javascript changes makes no sense as the element checking behavior does not require an ID 

```js
var mail = document.getElementById("login-user-email");
var pass = document.getElementById("login-user-password");
```
so this makes no sense as `val` contains the element.

And it still does not work

bildschirmfoto 2017-09-12 um 08 17 05

**Comment by dev7ch** [comment]
i am pretty sure ist an compiling issue, after compile i get this as expected: 

image


But i get also an Angular Error (white screen) after login:

image


Its not related to the login.js, its this issue still persist?

**Comment by nadar** [comment]
make sure you have run composer update in the admin and cms modules.

**Comment by TheMaaarc** [comment]
webkit-autofill and moz-autofill are pretty buggy. For example chrome handles type text and password differently, that's why the password label only floats to the top if you click **anywhere** on the page - until you do that the password field is considered empty by chrome.
I suggest we stop with that floating label stuff and just write the label above the input.

**Comment by dev7ch** [comment]
@nadar ok thanks, composer update inside admin module fixed copmile issue;

@marc, ok yes, i ve noticed its kind of pretty buggy. I ll keep this in mind for the future but for now i hope the implementation works a bit more reliable then before( testet in Firefox, Chrom, Opera, Safra - latest).

@marc, @nadra is the the floating autofill still not working for you?

**Comment by nadar** [comment]
i am not sure, i have to rewrite all your code first. 

> @dev7ch i just cleanup your code and the logic inside your javascript changes makes no sense as the element checking behavior does not require an ID 
```js
var mail = document.getElementById("login-user-email");
var pass = document.getElementById("login-user-password");
```
> so this makes no sense as `val` contains the element.

File: modules/admin/src/views/layouts/nosession.php (hunk 1 above)
Comment (dev7ch) [reply modules/admin/src/views/layouts/nosession.php]: yes, but we need to run this function in window on load context.

Comment (nadar) [reply modules/admin/src/views/layouts/nosession.php]: we did, directly inside the login.js

**Pull request merged**